<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edição de dados</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imds/diamond.png</normaloff>imds/diamond.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 175, 202);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1061</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edição de produtos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Código</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>132</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>518</x>
      <y>131</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Marca</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>132</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tamanho</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>132</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Preço de compra</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>746</x>
      <y>131</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Preço de venda</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_cod">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}

</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_nome">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_desc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_marca">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_tamanho">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_precv">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
    <property name="inputMask">
     <string>R$ 9999.99</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prod_precc">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: rgb(255, 255, 255);
border: 1px solid #ffffff;

}

QLineEdit::focus {
border: 2px solid #000000;
}
</string>
    </property>
    <property name="inputMask">
     <string>R$ 9999.99</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prod_salvar">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border-radius: 8px;

}

QPushButton::hover {

	color: rgb(255, 255, 255);
	border: 1px solid #ffffff;

}</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prod_voltar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border-radius: 8px;

}

QPushButton::hover {

	color: rgb(255, 255, 255);
	border: 1px solid #ffffff;

}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
